src/api.rs

```diff
@@ -1722,7 +1728,13 @@ impl ApiResponse {
     /// non okay response codes into errors.
     pub fn into_result(self) -> ApiResult<Self> {
         if let Some(ref body) = self.body {
-            debug!("body: {}", String::from_utf8_lossy(body));
+            let body = String::from_utf8_lossy(body);
+            debug!("body: {}", body);
+
+            // Internal helper for making it easier to write integration tests.
+            if let Ok(dir) = env::var("SENTRY_DUMP") {
```

Comment:
do we need to document `SENTRY_DUMP`? Also maybe we make the env variable more specific to sentry cli?

Reply:
I'd not do that tbh. It's only a helper for me, to make writing tests for all existing commands easier. I might remove it in the future entirely. Added better comment, renamed it and handled errors.